src/components/SyncServiceSignIn/index.js

```diff
@@ -35,7 +35,7 @@ function WebDAVForm() {
             persistField('webdavEndpoint', url);
             persistField('webdavUsername', username);
             persistField('webdavPassword', password);
-            window.location = window.location.origin + '/';
+            window.location = window.location.origin + process.env.PUBLIC_URL;
```

Comment:
How does it behave if PUBLIC_URL is undefined instead of `/`?

Reply:
Ah, good catch, thanks. It just turns into "" so the trailing slash would be missing. It looks like trailing slashes are removed from the `PUBLIC_URL` anyway (for both the `homepage` method and directly specifying `PUBLIC_URL`) so I'll simply re-add the `+ '/'` after the `PUBLIC_URL`.

Reply:
Alright, but does it also work for `export PUBLIC_URL='/'` (which should be allowed IMO)? Sorry for that annoyance; I also don't have a solution by hand right now...

Reply:
Yes I also tested that, this behaves the same as the other cases i.e. the slash is removed and `PUBLIC_URL` ends up as "" which is what we want.